### ISSUE TITLE
Reason for not using amodal image for content completion training

### ISSUE THREAD

**Issue by dqj5182** [Kaihua-Chen/diffusion-vas#7]
Always thank you for your responses. They helped me a lot with understanding your work.

I have another question regarding training process of Diffusion-VAS for Content Completion stage.
In your training code, it seems that you are appending synthetic occluder patch on top of the original image. But when I read the paper of [SAIL-VOS](https://openaccess.thecvf.com/content_CVPR_2019/papers/Hu_SAIL-VOS_Semantic_Amodal_Instance_Level_Video_Object_Segmentation_-_A_CVPR_2019_paper.pdf), it seems that the dataset amodal image. What I mean by amodal image is the RGB values for occluded regions that corresponds to amodal masks. I may be mistaken that SAIL-VOS provides such RGB values. But if they provide such RGB values, why not use them for supervision?

**Comment by dqj5182** [comment]
Thank you for your quick response!
